list-module-callers/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "scripts": {
+    "build": "ncc build src/index.ts --license LICENSE",
+    "test": "vitest"
+  },
+  "dependencies": {
+    "@actions/core": "1.10.1"
+  },
+  "devDependencies": {
+    "@types/node": "20.11.8",
+    "@vercel/ncc": "0.38.1",
+    "typescript": "5.3.3",
+    "vitest": "^1.2.2"
```

Comment:
When I run `npm t`, I have to input something.

```console
$ npm t

> test
> vitest


 DEV  v1.2.2 /Users/shunsukesuzuki/repos/src/github.com/suzuki-shunsuke/tfaction/list-module-callers

 ✓ test/lib.test.ts (2)
   ✓ resolveRelativeCallTree (1)
     ✓ resolves relative paths in terraform-config-inspect
   ✓ buildCallerToCallers (1)
     ✓ creates a map from callee to its direct callers and transitive callers

 Test Files  1 passed (1)
      Tests  2 passed (2)
   Start at  10:14:37
   Duration  94ms (transform 17ms, setup 0ms, collect 14ms, tests 1ms, environment 0ms, prepare 24ms)


 PASS  Waiting for file changes...
       press h to show help, press q to quit
```

Reply:
That is because Vitest runs test in watch mode by default, while Jest need `--watchAll` option.

Reply:
I see. After replacing Vitest with Jest, `npm t` command worked without any interruption.

```console
$ npm t

> test
> jest

 PASS  test/lib.test.ts
  resolveRelativeCallTree
    ✓ resolves relative paths in terraform-config-inspect (1 ms)
  buildCallerToCallers
    ✓ creates a map from callee to its direct callers and transitive callers (1 ms)

Test Suites: 1 passed, 1 total
Tests:       2 passed, 2 total
Snapshots:   0 total
Time:        0.543 s
Ran all test suites.
```